Scripts/gen_Digitize.sh

```diff
@@ -18,7 +19,11 @@ digitype=$6
 digout=${primary}${digitype}
 dbpurpose=$2_$7
 dbver=$8
-#
+#bfield="Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
+if [[ $# -eq 9 ]]; then
+  bfield="Offline/Mu2eG4/geom/$9"
```

Comment:
Might be a little more standard to not assume which directory the file is in and require the full path relative to the standard data path.  If you choose to not make this change, the relevant comment at the top of the file could make this choice more clear.

Reply:
I'd prefer to leave this.  We've adopted that directory as the official source of BField maps, someone can add whatever maps they want to their clone of Offline to test, so it's not a restriction.  The command line for gen_*.sh are already too long/complicated.